src/Illuminate/Foundation/Http/Middleware/PreventRequestsDuringMaintenance.php

```diff
@@ -23,15 +24,21 @@ class PreventRequestsDuringMaintenance
      */
     protected $except = [];
 
+    /**
+     * @var MaintenanceMode
+     */
+    private $maintenanceMode;
+
     /**
      * Create a new middleware instance.
      *
      * @param  \Illuminate\Contracts\Foundation\Application  $app
      * @return void
      */
-    public function __construct(Application $app)
+    public function __construct(Application $app, MaintenanceMode $maintenanceMode)
```

Comment:
Adding a new dependency here is a breaking change unfortunately.

Reply:
@driesvints You're right. It would for example break implementation if people have extended the class. Should I point the update to the Laravel 9 branch instead?